eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -187,6 +187,10 @@ function ($v) {
                                         ->defaultValue(5)
                                         ->info('Default version archive limit (0-50), only enforced on publish, not on un-publish.')
                                     ->end()
+                                    ->booleanNode('remove_archived_versions_on_publish')
+                                        ->defaultTrue()
+                                        ->info('Disabling and scheduling ezplatform:content:cleanup-versions command may speed up publishing request time')
```

Comment:
Info should describe what this option does.

```suggestion
                                        ->info('Enables automatic removal of archived versions when publishing, at the cost of performance. "ezplatform:content:cleanup-versions" command should be used to perform this task instead if this option is set to false.')
```